doc/api/readline.md

```diff
@@ -527,11 +528,113 @@ rl.on('line', (line) => {
 });
 ```
 
+## TTY keybindings
+
+<table>
+  <tr>
+    <th>Keybindings</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>ctrl+shift+backspace</code></td>
+    <td>delete line left</td>
```

Comment:
Not a must but I personally would prefer the descriptions all to start with a upper case character.

Reply:
done.